connectors/src/connectors/intercom/temporal/workflows.ts

```diff
@@ -52,6 +55,11 @@ export async function intercomSyncWorkflow({
 }) {
   await saveIntercomConnectorStartSync({ connectorId });
 
+  // Add folder node for teams
+  await upsertIntercomTeamsFolderActivity({
```

Comment:
exceptionally, just this once, we could discuss moving that into the syncTeamOnly activity
To be discussed with @overmode @philipperolet @aubin-tchoi vs the difficulty of stopping / restarting all intercom workflows for this change

Reply:
cc https://github.com/dust-tt/dust/pull/9469#discussion_r1893885581 I think we can get away with batch stop + batch resume